src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    String getPrintableString();
```

Comment:
Based on the diagram in the LO, `getPrintableString()` should be a public method (: 
Default modifier (no modifier) is `package-private` in Java, should not be confused with `private`, `protected` and `public`.